samples/air_routes/readme.md

```diff
@@ -1,19 +1,36 @@
 # Air Routes Sample
 
-This sample contains example Air Routes data imported from
-[here](https://github.com/krlawrence/graph/blob/main/sample-data/air-routes-latest.graphml).
+| Data Source  | Query Language | Database         |
+| ------------ | -------------- | ---------------- |
+| [Air Routes] | [Gremlin]      | [Gremlin Server] |
```

Comment:
I'm going to reorder these so that it's database, then query language, then data source.